lnp2p/peermgr.go

```diff
@@ -449,16 +449,16 @@ func (pm *PeerManager) SetupFunctionalMessageHandling() {
 	mp.DefineMessage(MsgidResponse, ParseRespMessage, func(p *Peer, m Message) error {
 		rm, ok := m.(respmsg)
 		if !ok {
-			return fmt.Errorf("bad message type")
+			return fmt.Errorf("bad peercall message type (resp)")
 		}
 		return pm.crouter.processReturnResponse(p, rm)
 	})
 
 	// Register error handling.
-	mp.DefineMessage(MsgidResponse, ParseRespMessage, func(p *Peer, m Message) error {
+	mp.DefineMessage(MsgidError, ParseErrMessage, func(p *Peer, m Message) error {
```

Comment:
Also this mistake was annoying too, because I just copy/pasted the above block and didn't think hard enough about it.